bruno/collections/Rafiki/scripts.js

```diff
@@ -127,7 +129,9 @@ const scripts = {
     bru.setEnvVar('tokenId', body?.access_token?.manage.split('/').pop())
   },
 
-  getWalletAddressId: async function (host, publicName, varName) {
+  loadWalletAddressIdsIntoVariables: async function () {
+    const requestUrl = this.resolveTemplateVariables(req.url)
```

Comment:
now, we just load up all of the walletAddressIds into variables for the respective Admin API that was called